pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/globalcomputeforwardingrule/_generated_object_globalcomputeforwardingrule.golden.yaml

```diff
@@ -37,7 +22,6 @@ spec:
   networkRef:
     name: default
   portRange: "80"
-  resourceID: computeglobalforwardingrule-${uniqueId}
```

Comment:
So I think we do need to keep setting spec.resourceID, for now, as the resource is a beta resource.

Reply:
With `state-into-spec: Absent` (1.122 and 1.123), only users will set this `spec.ResourceID` field. Do you mean we want to have the test coverage here that users use the `spec.resourceID` as the primary name rather than the `metadata.name`?  
I see in this example the `metadata.name` and the  `spec.resourceID` are set to the same value, which does not check if `spec.resourceID` is used.

Reply:
Hmmm... maybe we don't then :-)  I was worried about compatibility, but as it's not server-generated I don't think it will trip anyone up!